.github/workflows/snapshot-nhsn-data.yml

```diff
@@ -35,9 +35,9 @@ jobs:
 
       - name: Snapshot NHSN data
         run: Rscript -e "nhsn_data <- RSocrata::read.socrata('https://data.cdc.gov/resource/mpgq-jmmr.csv');
-                         nhsn_data <- nhsn_data[c("jurisdiction", "weekendingdate", "totalconfflunewadm", "totalconfc19newadm")];
+                         nhsn_data <- nhsn_data[c('jurisdiction', 'weekendingdate', 'totalconfflunewadm', 'totalconfc19newadm')];
                          nhsn_data$weekendingdate <- substr(nhsn_data$weekendingdate, 1, 10);
-                         colnames(nhsn_data) <- c("Geographic aggregation", "Week Ending Date", "Total Influenza Admissions", "Total COVID-19 Admissions");
+                         colnames(nhsn_data) <- c('Geographic aggregation', 'Week Ending Date', 'Total Influenza Admissions", "Total COVID-19 Admissions');
```

Comment:
missed a couple on this line - the second- and third-to-last

Reply:
thanks -- i made this change.  will go ahead and merge.